src/main/play-doc/general/MigrationGuide.md

```diff
@@ -33,18 +33,333 @@ In additional to disabling Elasticsearch via configuration, a service lookup mus
+
+
+##### A note on system, system version, and endpoint label
+
+Generating a customised HAProxy configuration will rely on the [[ifAcl|DynamicProxyConfiguration#ifAcl]] directive which accepts `system`, `systemVersion`, and `endpointLabel` as input argument.
+
+When declaring bundle settings via `build.sbt`, unless declared explicitly, the `system` and `systemVersion` settings may fallback to a default value as described by [sbt-conductr settings](https://github.com/typesafehub/sbt-conductr#bundle-settings).
+
+
+##### Customised port number
+
+Given the following `build.sbt` entry:
+
+```
+name := "my-bundle"
+BundleKeys.compatibilityVersion := "1.1"
+BundleKeys.endpoints := Map(
+  "endpoint-label" -> Endpoint("http", 0, Set(URI("http://:6789/service-name")))
+)
+
+```
+
+Or the following `bundle.conf` entry:
+
+```
+    system             = "my-bundle"
+    systemVersion      = "1.1"
+    endpoints          = {
+      "endpoint-label" = {
+        bind-protocol  = "http"
+        bind-port      = 0
+        services       = ["http://:6789/service-name"]
+      }
+    }
+```
+
+The following customised template will provide the equivalent proxy mapping:
+
+```
+{{#eachAcls bundles defaultHttpPort=9443}}
+  {{#ifAcl 'my-bundle' '1.1' 'endpoint-label'}}
+frontend my_www_frontend
+  bind {{haproxyHost}}:6789
+  mode http
+  acl my_www_path_match path_beg /service-name
+  use_backend my_www_backend if my_www_path_match
+
+backend my_www_backend
+  mode http
+  reqrep ^([^\ :]*)\ /service-name/?(.*) \1\ /\2
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+{{/eachAcls}}
+
+```
+
+##### Host header match
+
+Given the following `build.sbt` entry:
+
+```
+name := "my-bundle"
+BundleKeys.compatibilityVersion := "1.1"
+BundleKeys.endpoints := Map(
+  "endpoint-label" -> Endpoint("http", 0, Set(URI("http://www.acme.com")))
+)
+
+```
+
+Or the following `bundle.conf` entry:
+
+```
+    system             = "my-bundle"
+    systemVersion      = "1.1"
+    endpoints          = {
+      "endpoint-label" = {
+        bind-protocol  = "http"
+        bind-port      = 0
+        services       = ["http://www.acme.com"]
+      }
+    }
+```
+
+The following customised template will provide the equivalent proxy mapping:
+
+```
+{{#eachAcls bundles defaultHttpPort=9443}}
+  {{#ifAcl 'my-bundle' '1.1' 'endpoint-label'}}
+frontend my_www_frontend
+  bind {{haproxyHost}}:80
+  mode http
+  acl my_www_host_match hdr(host) -i www.acme.com
+  use_backend my_www_backend if my_www_host_match
+
+backend my_www_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+{{/eachAcls}}
+
+```
+
+
+### Migrating TCP-based services
+
+TCP based service can be migrated by supplying its [[request acl|RequestAclConfiguration]] equivalent.
```

Comment:
[[request ACL|AclConfiguration]] instead of [[request acl|RequestAclConfiguration]]